lights/light_regulator/LampRegulator.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace Lamp_Regulator 
+{
+    class Regulator
+    {
+        private byte[] _buffer = new byte[1024];
+        private List<Socket> _clientSockets = new List<Socket>();
+        private Socket _serverSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+        private int BACKLOG;
+        private int PORT;
+
+
+        public Regulator(int port, int backlog)
+        {
+            BACKLOG = backlog;
+            PORT = port;
+        }
+
+        public void StartRegulator()
```

Comment:
nie jestem zbytnim fanem takich metod, które wywołuja tylko jedna metodę w sobie, bo nie wydaje mi się, że poprawiaja czytelność a tak przy czytaniu kodu muszę szukać gdzie jest metoda `SetupServer()` i dopiero wiem co się dzieje w regulatorze. Ale jeżeli wg ciebie to jest ok to nie naciskam